app/controllers/application_controller.rb

```diff
@@ -4,11 +4,17 @@ class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
   before_action :authenticate_user!
   before_action :set_paper_trail_whodunnit
+  before_action :prepare_exception_notifier
```

Comment:
Should this be a `before_action` or `before_filter`? I'm looking at the documentation and you've used `before_action`. Why this change?

Reply:
`before_filter` is no longer supported in rails.